### PR TITLE
Added the "Assessment" dropdowns

### PR DESCRIPTION
**Pull request by s775518**
Added the "Assessment" dropdowns

### CODE REVIEW

File: custom_models/mental_health/models/mental_health_notes.py [hunk 1]
```diff
@@ -11,3 +11,89 @@ class MentalHealthNotes(models.Model):
     icd10_codes = fields.Selection(string='ICD10 Code', required=False, copy=False,
                                    selection=[('f40.00', 'F40.00 - Agoraphobia unspecified'),
                                               ('f40.01', 'F40.01 - Agoraphobia with panic disorder')])
+    thought_process = fields.Selection(string='Thought Process', copy=False,
```
Comment (DiamondDagger590) [inline custom_models/mental_health/models/mental_health_notes.py]: Should this field be required if a "normal" thought process is an option?

Comment (DiamondDagger590) [reply custom_models/mental_health/models/mental_health_notes.py]: Same goes for most of these fields tbh

Comment (s775518) [reply custom_models/mental_health/models/mental_health_notes.py]: I agree about making these fields required, and I think a number of them should also default to the neutral choice. That will be convenient for the user. @DiamondDagger590

Comment (DiamondDagger590) [reply custom_models/mental_health/models/mental_health_notes.py]: I agree, it all looks good from a QA standpoint, but I think we should get the default values set before it gets merged.

Comment (s775518) [reply custom_models/mental_health/models/mental_health_notes.py]: I set the default values and made the fields required. QA anybody?

File: custom_models/mental_health/models/mental_health_notes.py [hunk 2]
```diff
@@ -11,4 +11,102 @@ class MentalHealthNotes(models.Model):
...
+                                         ('slowed', 'Slowed'),
+                                         ('confident', 'Confident'),
+                                         ('energetic', 'Energetic'),
+                                         ('nervous', 'Nervous'),
+                                         ('fidgety', 'Fidgety')],
+                              default='normal',
+                              help='Select the affect of the client')
+    symptom_change = fields.Selection(string='Change in Symptom Severity', required=True, copy=False,
+                                      selection=[('resolved', 'Resolved'),
+                                                 ('much improved', 'Much improved'),
```
Comment (DiamondDagger590) [inline custom_models/mental_health/models/mental_health_notes.py]: Can we have the naming of the key be standardized to "much_improved" where any space is denoted with a "_"?

**Pull request merged**